app/controllers/admin/edition_workflow_controller.rb

```diff
@@ -65,6 +65,10 @@ def reject
                 notice: "Document rejected; please explain why in an internal note"
   end
 
+  def confirm_publish
+    redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ") and return unless @edition.valid?(:publish)
```

Comment:
Would this be a bit more readable if it wasn't a one-liner? I _think_ I know what it does... but I'd like to be more confident

Reply:
Yes I agree, which do you think is more clear?
```
    if @edition.valid?(:publish)
      nil
    else
       redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ")
     end
```
or 
```
    unless @edition.valid?(:publish)
      redirect_to admin_edition_path(@edition), alert: @edition.errors[:base].join(". ")
    end
```

or something else?

Reply:
I like the unless